docs/docs/reference/02-yaml-syntax.md

```diff
@@ -23,59 +23,402 @@ hardware:
+
+```yaml
+call_service:
+  service: <service_name>
+  component: <component_name>
+  payload:
+    foo: "some content"
+    bar: [ x, y, z ]
+    baz:
+      a: 1
+      b: 2
+```
+
+#### Load or unload a hardware interface
+
+Load and initialize a hardware interface.
+
+```yaml
+load:
+  hardware: <hardware_name>
+```
+
+Unload and destroy a hardware interface.
+
+```yaml
+unload:
+  hardware: <hardware_name>
+```
+
+#### Load or unload a controller
+
+```yaml
+load:
+  hardware: <hardware_name>
+  controller: <controller_name>
+
+unload:
+  hardware: <hardware_name>
+  controller: <controller_name>
+```
+
+Use a list to load or unload multiple controllers from a single predicate.
+
+```yaml
+load:
+  - hardware: <hardware_name>
+    controller: controller_a
+  - hardware: <hardware_name>
+    controller: controller_b
+```
+
+#### Activate or deactivate a controller
+
+Use the `switch_controllers` event to list the controllers to be activated or deactivated for a specific hardware
+interface.
+
+```yaml
+switch_controllers:
+  hardware: <hardware_name>
+  activate: [ <controller_name>, <controller_name> ]
+  deactivate: [ "controller_three", "controller_four" ] 
+```
+
+:::note
+
+A controller must be loaded before it can be activated, and must be deactivated before it can be unloaded.
+
+:::
+
+#### Manage sequences
 
-The `on_start` keyword is reserved as a special event trigger when the application is launched.
-List the [events](#events) to trigger on startup (for example, to load components and hardware interfaces).
+Use the `sequence` event to either start, restart or abort a named [sequence](#sequences) in the application
+description. Use the respective `start`, `restart` or `abort` fields either individually or collectively.
 
 ```yaml
-on_start:
-  load:
-    - component: component_a
-    - component: component_b
-    - hardware: robot_a
+sequence:
+  start: <sequence_name>
+  restart: <sequence_name>
+  abort: <sequence_name>
 ```
 
-## Buttons
-
-Buttons are interactive elements in the Developer Interface UI. They are used to manually trigger state events when an
-application is running by clicking the trigger button in the application graph. Buttons have no effect on the
-application if the UI is not used.
+To manage multiple sequences with the same event trigger, use a list syntax.
 
 ```yaml
-buttons:
-  my_button:
-    position: ...
-    on_click:
-      ...
+sequence:
+  - start: "sequence_a"
+  - start: "sequence_b"
 ```
 
-### Position
+### Special event predicates
 
-The `position` field is used to define the desired location of the button in the application graph. It has two subfields
-defining the X and Y location, respectively.
+#### on_load
+
+The `on_load` predicate is provided by the state engine and set to true after the component
+has been loaded. Any events associated with the `on_load` predicate are handled once
+on instantiation of the node.
```

Comment:
nitpicking (or misreading -- excuse my English if so --). The second part reads a bit odd to me, I think a comma after `once` would fix the ambiguity in my head :P (or rephrasing).

Reply:
The suggested version sounds clearer to me, but I agree that even the first one works.